libs/blocks/merch-card/merch-card.js

```diff
@@ -0,0 +1,205 @@
+import { addBackgroundImg, addWrapper, addFooter } from '../card/cardUtils.js';
+import { decorateLinkAnalytics } from '../../martech/attributes.js';
+
+const {
+  miloLibs,
+  codeRoot,
+} = getConfig();
+const base = miloLibs || codeRoot;
+const SECURE_TRANSACTION_IMG = `<img class="secure-transaction-icon-img" loading="lazy" alt="secure-transaction-icon icon" src="${base}/blocks/merch-card/img/secure-transaction-icon.svg" height="12" width="15">`;
+
+const SEGMENT_BLADE = 'SegmentBlade';
+const SPECIAL_OFFERS = 'SpecialOffers';
+const PLANS_CARD = 'PlansCard';
+
+const getPodType = (styles) => {
+  const cardTypes = {
+    segment: SEGMENT_BLADE,
+    'special-offers': SPECIAL_OFFERS,
+    plans: PLANS_CARD,
+  };
+  const authoredType = styles?.find((style) => style in cardTypes);
+  return cardTypes[authoredType] || SEGMENT_BLADE;
+};
+
+const createDescription = (rows, cardType) => {
+  const descriptions = rows.slice(0, rows.length - 1);
+  const descriptionWrapper = createTag('div', { class: `consonant-${cardType}-description` });
+  descriptions?.forEach((description) => descriptionWrapper.appendChild(description));
+  return descriptionWrapper;
+};
+
+const createTitle = (titles, cardType) => {
+  const titleWrapper = createTag('div', { class: `consonant-${cardType}-title` });
+  titles?.forEach((title) => {
+    if (!title.id.includes('secure-transaction')) titleWrapper.appendChild(title);
+  });
+  return titleWrapper;
+};
+
+const decorateFooter = (el, cardType) => {
+  const cardFooter = el.querySelector('.consonant-CardFooter');
+  const decorateWithSecureTransactionSign = (secureTransaction) => {
+    const secureTransactionWrapper = createTag('div', { class: 'secure-transaction-wrapper' });
+    const label = createTag('span', { class: 'secure-transaction-label' }, secureTransaction?.textContent);
+    const secureElement = createTag('span', { class: 'secure-transaction-icon' }, SECURE_TRANSACTION_IMG);
+    secureTransactionWrapper.append(secureElement, label);
+    return secureTransactionWrapper;
+  };
+
+  const createCheckbox = (checkBoxText) => {
+    cardFooter.querySelector('hr')?.remove();
+    const container = createTag('label', { class: 'checkbox-container' });
+    const input = createTag('input', { id: 'alt-cta', type: 'checkbox' });
+    const checkmark = createTag('span', { class: 'checkmark' });
+    const label = createTag('span', { class: 'checkbox-label' }, checkBoxText.innerHTML);
+    container.append(input, checkmark, label);
+    return container;
+  };
+
+  const decorateAlternativeCta = () => {
+    const toggleButtonActiveState = (buttonToAdd, buttonToRemove) => {
+      buttonToAdd.classList.add('button--inactive');
+      buttonToRemove.classList.remove('button--inactive');
+    };
+    const cardFooterRow = el.querySelector('.consonant-CardFooter-row');
+    const secureTransactionSign = el.querySelector('h6[id^="secure-transaction"]');
```

Comment:
We probably need a variant for this rather then relying on the block to have an `<h6 id="Secure Transaction">` element... 
This could ruffle some feathers in localization

Reply:
Have you tried testing this in a different locale? Is this using a placeholder? 
I'm not sure how this is marked as 'resolved'?

Reply:
@VKniaz  - Any thoughts around removing the `h6` part of this selector. Keep it more authoring-proof since placeholders are often just typed w/ plain text.